internal/pkg/testutils/testutils.go

```diff
@@ -0,0 +1,25 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package testutils
+
+import (
+	"runtime"
+	"testing"
+
+	"github.com/elastic/elastic-agent/internal/pkg/agent/application/secret"
+	"github.com/elastic/elastic-agent/internal/pkg/agent/storage"
+	"github.com/elastic/elastic-agent/internal/pkg/agent/vault"
+)
+
+func InitStorage(t *testing.T) {
```

Comment:
Do we want all tests that use this to have the same storage location?

Reply:
This is used for the previous tests that needed access to the configuration storage and started to fail after introduction of encryption. Many of these tests are already touching the same configuration files paths as far as I can see in the code. This specific change introduces the key storage in order to have the encrypted storage to work.